models/base/manifest/default/snowplow_mobile_base_sessions_lifecycle_manifest.sql

```diff
@@ -3,10 +3,10 @@
     materialized=var("snowplow__incremental_materialization"),
     unique_key='session_id',
     upsert_date_key='start_tstamp',
-    partition_by = {
+    partition_by = snowplow_utils.get_partition_by(bigquery_partition_by={
       "field": "start_tstamp",
       "data_type": "timestamp"
-    },
+    }, databricks_partition_by='start_tstamp'),
```

Comment:
I know we do this in the web model but actually I think we should remove this. It's going to be quite slow since databricks isn't smart about partitioning and so will create individual partitions for every timestamp rather than aggregate to a date level.

We can either drop the partitioning, or we can add a DATE() column and then partition on that. WDYT?

Reply:
Obviously we can patch the same thing in web (at the same time as the spark update)